config/application.rb

```diff
@@ -7,16 +8,79 @@
 
 module CaseflowEfolder
   class Application < Rails::Application
-
     # Initialize configuration defaults for originally generated Rails version.
-    # config.load_defaults 5.1
-    
+    config.load_defaults 5.2
+
     # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
+    # Application configuration can go into files in config/initializers
+    # -- all .rb files in that directory are automatically loaded after loading
+    # the framework and any gems in your application.
 
-    config.download_filepath = Rails.root + "tmp/files"
+    #=======================================================================================
+    # Rails 5.0 default overrides
+    #---------------------------------------------------------------------------------------
+    
+    # Enable per-form CSRF tokens. Previous versions had false.
+    # Deafault as of 5.0: true
+    Rails.application.config.action_controller.per_form_csrf_tokens = false
 
+    # Enable origin-checking CSRF mitigation. Previous versions had false.
+    # Default as of 5.0: true
+    Rails.application.config.action_controller.forgery_protection_origin_check = false
+
+    # Make Ruby 2.4 preserve the timezone of the receiver when calling `to_time`.
+    # Previous versions had false.
+    # Default as of 5.0: true
+    ActiveSupport.to_time_preserves_timezone = false
+
+    # Require `belongs_to` associations by default. Previous versions had false.
+    # Default as of 5.0: true
+    Rails.application.config.active_record.belongs_to_required_by_default = false   
+
+
+    #=======================================================================================
+    # Rails 5.1 default overrides
+    #---------------------------------------------------------------------------------------
+    
+    # Make `form_with` generate non-remote forms.
+    # Default as of 5.1: true
```

Comment:
I think it's also worth noting here that the default for this setting changes back to `false` on version `6.1`. This will be relevant for us when we upgrade to `6.1`, as we can remove this override at that point when we adopt `config.load_defaults 6.1`.

https://guides.rubyonrails.org/configuring.html#config-action-view-form-with-generates-remote-forms



Starting with version | The default value is
-- | --
5.1 | true
6.1 | false



```suggestion
    # Default as of 5.1: true
    # Default as if 6.1: false
```